README.md

```diff
@@ -1,5 +1,53 @@
-# Heroku Cloud Native PHP Buildpack
+# Heroku Cloud Native Buildpack: PHP
 
-[![CI](https://github.com/heroku/buildpacks-php/actions/workflows/ci.yml/badge.svg)](https://github.com/heroku/buildpacks-php/actions/workflows/ci.yml)
+[![Cloud Native Buildpacks Registry: heroku/php][registry-badge]][registry-url]
+[![CI on Github Actions: heroku/php][ci-badge]][ci-url]
 
-Heroku's official [Cloud Native Buildpack](https://buildpacks.io) for the PHP ecosystem.
+![Heroku Cloud Native Buildpack: heroku/php][cnb-banner]
+
+`heroku/php` is the [Heroku Cloud Native Buildpack][heroku-buildpacks]
+for PHP applications. It builds PHP application source code into application images with
+minimal configuration.
+
+> [!IMPORTANT]
+> This is a [Cloud Native Buildpack][cnb], and is a component of the [Heroku Cloud Native Buildpacks][heroku-buildpacks] project, which is in preview. If you are instead looking for the Heroku Classic Buildpack for PHP (for use on the Heroku platform), you may find it [here][classic-buildpack].
+
+## Usage
+
+> [!NOTE]
+> Before getting started, ensure you have the `pack` CLI installed. Installation instructions are available [here][pack-install].
+
+To build a PHP application codebase into a production image:
+
+```bash
+$ cd ~/workdir/sample-php-app
+$ pack build sample-app --builder heroku/builder:22
+```
+
+Then run the image:
+```bash
+docker run --rm -it -e "PORT=8080" -p 8080:8080 sample-app
+```
+
+## Application Requirements
+
+TODO
+
+## Configuration
+
+TODO
+
```

Comment:
Remove requirements and configuration sections for now (@dzuelke can then update/add these back, if necessary, in a separate PR when he's back)

```suggestion
```